brainstormforce/ultimate-addons-for-gutenberg
# Fix: Blockquote - Tweet content not shown in tweet box when double and single quotes are used.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/iHKt2gD8SSQ/d9be5753-3c67-44f0-84b2-e99dd6642598?view=159c7a1b-ad81-488e-a0f2-de291530940f&t=attributes

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Blockquote.
- Add single and double quotes around the string-like https://share.getcloudapp.com/nOu55mAE
- Update and click on the Tweet button and tweet content is not visible in the tweet box.
- Switch to `blockqoutes-single-double-qoutes-issue` and check is it resolved or not.

### Checklist:
